packages/workflow-cmd/src/commands/update.js

```diff
@@ -0,0 +1,178 @@
+/* eslint-env node */
+/* eslint-disable no-console */
+import execa from 'execa';
+import ncu from 'npm-check-updates';
+import { join } from 'path';
+import fs from 'fs';
+import prompt from 'prompt';
+import { promisify } from 'util';
+import readPkgUp from 'read-pkg-up';
+
+import { baseFolder } from 'shared/env';
+import { which } from 'shared/shell';
+
+const promptGetAsync = promisify(prompt.get);
+
+export default async function update(args) {
+  if (args.force || (await noUncommitChanges(baseFolder))) {
+    if (args.force) {
+      console.log('Ignoring uncommited changes with `--force` flag');
+    }
+
+    await updateWorkflowCommand();
+    await updateWorkflowHome(args);
+  } else {
+    console.log('Uncommited changes was found in', baseFolder);
+    console.log('Either commit the changes or re-run this command with `--force`.');
+  }
+}
+
+async function noUncommitChanges(folder) {
+  if (which('git').code === 1) {
+    console.log('`git` command was not found.');
+    console.log(
+      `Install git and add a repository to  ${folder} or re-run this command with \`--force\``
```

Comment:
I feel like this is kind of being pushy about having workflow home as a git repo. Why not just go ahead if git is not used?

Reply:
Good idea :) changed the to just print a warning about disabling uncommited changes check.